firmware/controllers/core/main_loop.h

```diff
@@ -7,6 +7,7 @@ enum class LoopPeriod : uint8_t {
 	Period1000hz = 1 << 0,
 	Period500hz = 1 << 1,
 	Period250hz = 1 << 2,
+  	Period200hz = (1 << 2) + 1, // why not the plain 5?
```

Comment:
@FDSoftware note ``inline constexpr LoopPeriod& operator|=`` and ``inline constexpr bool operator&`` I would assume it's something to do with bitmap masks? or is that what you are trying to do here?

i.e. we have to be careful I would assume. Sounds like we need a unit test :)

Reply:
> @FDSoftware note `inline constexpr LoopPeriod& operator|=` and `inline constexpr bool operator&` I would assume it's something to do with bitmap masks? or is that what you are trying to do here?
> 

I saw the bitmasks but at the same time this enum ends up being an int8, the rest of the bitmasks also end in numbers that are equivalent to the amount of ms like the previous code, that's why it caused me some confusion

> i.e. we have to be careful I would assume. Sounds like we need a unit test :)

Reply:
@rusefillc after using tests from #7765, adding a non bit-shift value like 5ms to the enum breaks the logic (and the new tests) on makePeriodFlag, I am going to review the HPFP.PI test to update it to the new execution rate and revert the 200hz change

Reply:
@FDSoftware can we add a value between existing ones? Shift 50hz one bit?

Reply:
on `HpfpQuantity::calcPI` we have this code using the fast callback define
```c++
float possibleValue = m_pressureTarget_kPa - (engineConfiguration->hpfpTargetDecay *
			(FAST_CALLBACK_PERIOD_MS / 1000.));
```
![image](https://github.com/user-attachments/assets/f55ca20b-54c4-47fa-96de-37eb12cbd119)

The affected test cases are "Test pressure decay" and "Integral gain", which both depend on the execution time.

@racer-coder do you agree with the changes in the tests?

Reply:
> @FDSoftware can we add a value between existing ones? Shift 50hz one bit?

I think I can add one at the end like ` 1 << 4,` now I'm going to try it

Reply:
> It's a very very VERY good indication that if an enum has the values specified as `1 << n`, then it's a bit field, not an arbitrary value. Enums for internal use (ie, they don't go in the config) don't really have any other reason to specify values.

thx! I was a little confused about this enum thing.